test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py

```diff
@@ -8,6 +8,12 @@
 import os
 import sys
 
+import yaml
+from yaml.resolver import Resolver
+from yaml.constructor import SafeConstructor
+from yaml.error import MarkedYAMLError
+from _yaml import CParser
```

Comment:
Why `from _yaml` instead of `from yaml`?

Reply:
It needs to be imported from the compiled extension, i think (that's how it was when i wrote the code for ansible, at least.  That may have changed in the years since then)

Reply:
Just confirmed, CParser is only visible in the compiled extension so you have to import it from there.

Reply:
OK, I was looking at `CLoader`, which is available from `yaml` -- not `CParser`.